src/services/initializer-service.ts

```diff
@@ -112,6 +121,21 @@ export class InitializerService {
     this.filesys.createFileFromTemplate(['Dockerfile'], 'Dockerfile-template')
   }
 
+  public createDefaultAuxFiles(): void {
+    InitializerService.debug('creating auxfiles')
+
+    const replaceDict = { '%BUNDLENAME%': this.options.name }
+    const defaultYamls = ['mysql', 'postgresql', 'keycloak']
```

Comment:
not needed, just read all files in the `aux` folder so we don't need to modify this if we add other aux files

Reply:
done